common/client-libs/validator-client/src/nyxd/fee/mod.rs

```diff
@@ -144,29 +144,11 @@ mod sealed {
         }
     }
 
-    fn cosmos_decimal_inner_getter(val: &CosmosDecimal) -> u64 {
-        // haha, this code is so disgusting. I'll make a PR on cosmrs to slightly alleviate those issues...
-        // note: unwrap here is fine as the to_string is just returning a stringified u64 which, well, is a valid u64
-        val.to_string().parse().unwrap()
-    }
-
-    // at the time of writing it the current cosmrs' Decimal is extremely limited...
-    #[derive(Serialize, Deserialize)]
-    #[serde(remote = "CosmosDecimal")]
-    struct Decimal(#[serde(getter = "cosmos_decimal_inner_getter")] u64);
-
-    impl From<Decimal> for CosmosDecimal {
-        fn from(val: Decimal) -> Self {
-            val.0.into()
-        }
-    }
-
     #[derive(Serialize, Deserialize, Clone)]
     struct Coin {
         #[serde(with = "Denom")]
```

Comment:
iirc we could also remove that special serde annotation like you've done with Decimal